Makefile

```diff
@@ -208,20 +161,21 @@ lint-js:
         create-db \
         apply-migrations \
         reset-db \
-        setup-pyenv \
-        setup-git-config \
```

Comment:
I've reordered these so I can visually compare the make targets in the file and making sure that we're not missing any of them.